equinix-openapi-metal/api/openapi.yaml

```diff
@@ -19673,7 +19662,7 @@ components:
           metal_gateway:
             gateway_address: gateway_address
             updated_at: 2000-01-23T04:56:07.000+00:00
-            vlan: 0.8008281904610115
+            vlan: 1.4658129805029452
```

Comment:
This seems to be the wrong kind of numeric type.

Reply:
https://github.com/equinix-labs/metal-java/pull/37/files#diff-d0c21defbc32ac215a9be5981d47adb797aa8a5e5841b08f4f8b9d36d7324b78R15500 should be `integer` https://swagger.io/docs/specification/data-models/data-types/#numbers